Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -218,7 +226,15 @@ if [[ -f /.dockerenv ]] || grep -Eq '(lxc|docker)' /proc/1/cgroup ; then
 fi
 echo "Done!"
 
-SHELL_LOGIN=".profile"
```

Comment:
I don't think this is right, the advantage of the .profile is that it works with other shell system and not only bash !

Reply:
Hi there, thanks for the reply.

The bash related lines are there because this is a bash script (see line 1: #!/bin/bash) so it would be logical to support bash. Additionally, to my knowledge 99.99% of Debian-based systems use bash.

I agree that profile would be better if the first line read "#!/bin/sh".